tests/ffi/tests.cc

```diff
@@ -598,9 +598,26 @@ extern "C" const char *cxx_run_test() noexcept {
   r_take_unique_ptr(std::unique_ptr<C>(new C{2020}));
   r_take_ref_c(C{2020});
   r_take_str(rust::Str("2020"));
+  r_take_empty_str(rust::Str(""));
```

Comment:
Added "empty" tests for `std::string_view` and figured we might as well add tests for `rust::str` and 'rust::String`